core/src/test/resources/data/responses/invalids/attacks/response_with_spoofed_response_signature.xml

```diff
@@ -0,0 +1,35 @@
+<samlp:Response xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion" ID="pfx1ff1da4a-010e-428c-16eb-75f31f16567a" Version="2.0" IssueInstant="2014-02-19T01:37:01Z" Destination="http://localhost:8080/java-saml-jspsample/acs.jsp" InResponseTo="ONELOGIN_5fe9d6e499b2f0913206aab3f7191729049bb807">
+    <saml:Issuer>https://pitbulk.no-ip.org/simplesaml/saml2/idp/metadata.php</saml:Issuer>
+    <samlp:Extensions>
+    	<saml:Response ID="fakeSignature">
+    		<ds:Signature xmlns:ds="http://www.w3.org/2000/09/xmldsig#">
```

Comment:
This signature was found by `processSignedElements`, and the validation considered the Response signed, but subsequently wasn't validated by `validateSign`, because that one was looking for the Signature in the expected place. `validateSign` still passed, because the Assertion signature was okay, and it only required one of the signatures to be valid.